frontend/js/src/utils/APIService.ts

```diff
@@ -931,6 +931,27 @@ export default class APIService {
     return response.json();
   };
 
+  submitMbidMapping = async (
+    userToken: string,
+    recordingMSID: string,
+    recordingMBID: string
+  ): Promise<{ status: string; data: PinnedRecording }> => {
```

Comment:
wrong type, but if we're not using the returned data then we could just remove this from the returned value